UPGRADE-2.0.md

```diff
@@ -0,0 +1,28 @@
+UPGRADE FROM 1.x to 2.0
+========================
+
+GridFS
+------
+
+ * The `md5` is no longer calculated when a file is uploaded to GridFS.
+   Applications that require a file digest should implement it outside GridFS
+   and store in metadata.
+
+   ```php
+   $hash = hash_file('sha256', $filename);
+   $bucket->openDownloadStream($fileId, ['metadata' => ['hash' => $hash]]);
```

Comment:
I think all of these examples should be using `openUploadStream()` since they're demonstrating uploads. `openDownloadStream()` only takes a single `$id` argument.

Applies to the before/after examples in the subsequent bullet, too.

Reply:
Updated.